apps/web/components/booking/BookingListItem.tsx

```diff
@@ -654,45 +672,255 @@ const FirstAttendee = ({
 type AttendeeProps = {
   name?: string;
   email: string;
+  id: number;
+  noShow: boolean;
+};
+
+type NoShowProps = {
+  bookingUid: string;
+  isPast: boolean;
 };
 
-const Attendee = ({ email, name }: AttendeeProps) => {
+const Attendee = (attendeeProps: AttendeeProps & NoShowProps) => {
+  const { email, name, bookingUid, isPast, noShow: noShowAttendee } = attendeeProps;
+  const { t } = useLocale();
+
+  const [noShow, setNoShow] = useState(noShowAttendee);
+  const [openDropdown, setOpenDropdown] = useState(false);
+
+  const noShowMutation = trpc.viewer.public.noShow.useMutation({
+    onSuccess: async () => {
+      noShow
+        ? showToast(
+            t("x_marked_as_no_show", {
+              x: email,
+            }),
+            "success"
+          )
+        : showToast(
+            t("x_unmarked_as_no_show", {
+              x: email,
+            }),
+            "success"
+          );
```

Comment:
```suggestion
  showToast(
            t(noShow ? "x_marked_as_no_show" : "x_unmarked_as_no_show", {
              x: email,
            }),
            "success"
          )
```

NIT: maybe something like this is a bit easier to understand